tools/sync/command.py

```diff
@@ -26,6 +26,13 @@
 import subprocess
 import string
 import threading
+import time
+
+
+class TimeoutException(Exception):
+    """
+    Exception returned when command exceeded its timeout.
+    """
```

Comment:
\+ `pass`

Reply:
I think the block comment does the same service however for readability it does not hurt so adding.